src/main/resources/application.properties

```diff
@@ -70,3 +70,10 @@ audit.history.months=6
 audit.history.endpoint=${pttg.audit.url}/history
 audit.archive.endpoint=${pttg.audit.url}/archive
 
+#
+# Service timeouts
+#
+timeouts.hmrc-service.read-ms=120000
```

Comment:
This isn't the value on the Jira

> ip api - ip hmrc: 110s

Reply:
Yes good spot, got it mixed up with the UI timeouts.